.evergreen/evergreen.yml.in

```diff
@@ -917,6 +917,9 @@ functions:
           {
           export NODE_JS_VERSION=${node_js_version}
           source .evergreen/setup-env.sh
+          npm run evergreen-release bump
+          git add .
+          git commit --no-allow-empty -m "chore(release): bump to prepare for mongosh release"
```

Comment:
The idea here is that this will bump both "independent" and mongosh packages before the release draft and push it to main, so our branch is consistent with what we're releasing.
It might also make sense to only persist this after the release is published.

I am assuming this should work? Though not sure if the evergreen git setup has ability to push to main like this?